config/prod/s3-synapse-sync.yaml

```diff
@@ -8,7 +8,7 @@ dependencies:
   - "prod/htan-synapse-sync-kms-key.yaml"
 hooks:
   before_launch:
-    - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/s3-synapse-sync/master/s3-synapse-sync.yaml --create-dirs -o templates/remote/s3-synapse-sync.yaml"
+    - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/s3-synapse-sync/v1.1/s3-synapse-sync.yaml --create-dirs -o templates/remote/s3-synapse-sync.yaml"
```

Comment:
Your tag doesn't seem to match the version that's in the cfn template, https://github.com/Sage-Bionetworks/s3-synapse-sync/blob/master/template.yaml#L19

It would be better if it matches.  You tag seems to be ok though, so either change the version references in the template or redo your tags. 

here's the deployed file from your tag, https://bootstrap-awss3cloudformationbucket-19qromfd235z9.s3.amazonaws.com/index.html#s3-synapse-sync/v1.1/